sidebars.ts

```diff
@@ -2401,6 +2401,7 @@ integrations: [
           'integrations/sumo-apps/infrequent-data-tier',
           'integrations/sumo-apps/log-analysis-quickstart',
           'integrations/sumo-apps/security-analytics',
+          // 'integrations/sumo-apps/sumo-logic-sample-data-astronomy',
```

Comment:
@ishaanahuja29 , to confirm, this app is GA, not beta? If GA, you'd need to apply this change.

```suggestion
          'integrations/sumo-apps/sumo-logic-sample-data-astronomy',
```